components/browser/menu/src/main/java/mozilla/components/browser/menu/BrowserMenu.kt

```diff
@@ -61,6 +62,12 @@ class BrowserMenu internal constructor(
             adapter = this@BrowserMenu.adapter
         }
 
+        val recyclerView = view.findViewById<RecyclerView>(R.id.mozac_browser_menu_recyclerView)
+        recyclerView.minimumWidth = when (extras?.get("minimum_width")) {
+            null -> WindowManager.LayoutParams.WRAP_CONTENT
+            else -> extras["minimum_width"] as Int
+        }
```

Comment:
nit: We already find and manipulate the RecyclerView a couple of lines above (`menuList`). :)

Reply:
🤦‍♂ whoops.